package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "postinstall": "lerna bootstrap",
     "test": "lerna run test",
+    "test:e2e": "lerna run e2e:saucelabs --scope @wmde/wikit-vue-components",
```

Comment:
What made you go with the e2e label instead of browser test? In most code bases I know e2e tests are the kind that send http requests to the application webserver and inspect the response, but without a browser.

Reply:
The problem I encountered was that in CI we run test:* before the storybook commands, but for the cross browser tests we need a built and deployed storybook, so the e2e needs to run last. 
Removing `test:` was a quick solution. I suppose we can have it as `test:e2e` but then we need to change the lerna test to run all but the e2e.

Reply:
Ah, I wasn't asking about `e2e` vs `test:e2e`, but rather why `e2e`/`test:e2e` and not `test:browser`? In termbox for example `tests/edge-to-edge` contains non-browser e2e tests that make http requests against the server and expect certain responses, whereas ~~`tests/browser`~~ `tests/selenium` contain the actual browser tests. I think I saw a similar structure in a FUN (?) code base as well.

Not a big deal, though. It can be renamed later, if anyone trips over it.

Reply:
Oooh, I didn't actively name it `e2e`. It came from the [@vue/cli-plugin-e2e-nightwatch](https://www.npmjs.com/package/@vue/cli-plugin-e2e-nightwatch) plugin.